src/components/views/game/game/scoreannouncing/AnnouncementModal.scss

```diff
@@ -56,6 +57,9 @@
 
     .card-component {
       cursor: pointer;
+      &:hover {
+        transform: scale(100%);
```

Comment:
minor: maybe unset would be enough

Reply:
ah cool, I look up for this css line of code.

Reply:
pointer-event: none;